quic-client/src/nonblocking/quic_client.rs

```diff
@@ -106,6 +107,11 @@ impl QuicLazyInitializedEndpoint {
 
         let mut crypto = rustls::ClientConfig::builder()
             .with_safe_defaults()
+            .with_cipher_suites(&[
```

Comment:
Let's add this change here as well: https://github.com/solana-labs/solana/blob/0f16120b0342645a9ea93f490681ec3a0638d66b/streamer/src/quic.rs#L74

This will update the test to use the same client configuration.